### PR TITLE
Adding tipster and progressbar styles

### PR DESCRIPTION
**Pull request by birdy-**

### CODE REVIEW

File: src/bootstrap-enrichment/_tipster.scss [hunk 1]
```diff
@@ -0,0 +1,23 @@
+.tipster {
+  position: relative;
+  display: inline-block;
+  padding: 10px 20px;
+  background: #fff;
+  box-shadow: 0 0 5px #000;
```
Comment (adlenafane) [inline src/bootstrap-enrichment/_tipster.scss]: Is there a materialui shadow class matching? like `shadow-z-1` or similar?

File: src/bootstrap-enrichment/_tipster.scss [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  margin: 10px 10px 18px 10px;
+  font-size: 24px;
+  color: $brand-danger;
+  &:after {
+      display: block;
+      content: "";
+      box-shadow: -2px 2px 2px rgba(0,0,0,0.4);
+      border: 10px solid #fff;
+      border-color: transparent transparent #fff #fff;
+      transform: rotate(-45deg);
```
Comment (adlenafane) [inline src/bootstrap-enrichment/_tipster.scss]: No need to use a mixin for vendor prefixes?

**Comment by adlenafane** [comment]
LGTM :+1:

File: src/bootstrap-enrichment/_tipster.scss (hunk 2 above)
Comment (birdy-) [inline src/bootstrap-enrichment/_tipster.scss]: Nop since we use autoprefixer ?

File: src/bootstrap-enrichment/_tipster.scss (hunk 1 above)
Comment (birdy-) [inline src/bootstrap-enrichment/_tipster.scss]: Maybe, i'll check

File: src/bootstrap-enrichment/_tipster.scss (hunk 2 above)
Comment (adlenafane) [inline src/bootstrap-enrichment/_tipster.scss]: :ok_hand:

**Comment by adlenafane** [comment]
LGTM :+1:

**Comment by saury07** [comment]
LGTM, even if I have almost no idea what it does.

**Comment by adlenafane** [comment]
@saury07 You may ask for a screenshot

**Pull request merged**